packages/contracts-bedrock/foundry.toml

```
@@ -25,6 +25,8 @@ remappings = [
 extra_output = ['devdoc', 'userdoc', 'metadata', 'storageLayout']
 bytecode_hash = 'none'
 build_info_path = 'artifacts/build-info'
+evm_version = "cancun"
+ignored_error_codes = [2394, 5574, 3860]
```

This should work and is more readable ([source](https://github.com/foundry-rs/foundry/blob/bf5bfe0a3af293152f84f965e21856bc740aac0d/crates/config/src/error.rs#L152))
```suggestion
ignored_error_codes = ["transient-storage", "code-size", "init-code-size"]
```